packages/arb-token-bridge-ui/src/components/common/Button.tsx

```diff
@@ -28,9 +28,9 @@ export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
   variant: ButtonVariant
   loading?: boolean
   loadingProps?: ButtonLoadingProps
-  textLeft?: true
-  showArrow?: true
-  truncate?: false
+  textLeft?: boolean
+  showArrow?: boolean
+  truncate?: boolean
```

Comment:
had to update it because the type mismatch warning was polluting the dev console